src/ClientBundle/Resources/public/js/view/client_select.js

```diff
@@ -68,12 +70,18 @@ define(
 
                 if (!_.isUndefined(show)) {
                     if (true === show) {
-                        this.$('#client-select-container').show();
+                        clientSelectContainer.show();
                     } else {
-                        this.$('#client-select-container').hide();
+                        clientSelectContainer.hide();
                     }
                 } else {
-                    this.$('#client-select-container').toggle();
+                    clientSelectContainer.toggle();
+                }
+            },
+            selectDefaultClientContact: function () {
+                const clientSelectContainer = this.$('#client-select-container');
+                if (clientSelectContainer.find('input[type="checkbox"]').length === 1){
```

Comment:
`clientSelectContainer.find('input[type="checkbox"]')` can be assigned to a variable that can be re-used

Reply:
Thanks lot, good notes. I applied changes.